exp-player/addon/randomizers/pref-phys-pilot.js

```diff
@@ -344,12 +343,25 @@ function toFrames(frameId, eventVideos, BASE_DIR) {
                 BASE_DIR + 'stimuli/attention/',
                 'attentiongrabber'),
             sources: videoSourceObjs(
-		BASE_DIR + 'stimuli/' + features.eventType + '/',
-		e.fname, true),
+		        BASE_DIR + 'stimuli/' + features.eventType + '/',
+		        e.fname, true),
             altSources: videoSourceObjs(
                 BASE_DIR + 'stimuli/' + features.eventType + '/',
                 e.altName, true)
         };
+
+        // FOR PILOT ONLY: replace fall videos with calibration
+        if (e.compType === 'fall') {
+            returnFrame.sources = videoSourceObjs(
+                BASE_DIR + 'stimuli/attention/',
+                'calibration');
+            returnFrame.sources = videoSourceObjs(
```

Comment:
If you're not near a computer, I'll go ahead and make those changes myself...?

Before we go to production on this, I suggest trying it out on the staging server tonight (along with the sentry logging feature). That will avoid surprises when this goes to production.

Naive question, but I assume that `e.compType === 'fall'` is something that only happens in the pilot? (I ran through a sample study earlier on staging, and saw some items falling. I may be overinterpreting that)